### DISCUSSION TITLE
Para tirar o Erro na hora de conectar ao Banco de dados pela primeira vez...

### DISCUSSION THREAD

**Discussion by PatrickRebecchi** [balta-io/forum#565] [Tira Dúvidas] [answered]
na parte do connectionString ao invés de usar =  `const` string connectionString = "Server=localhost,1433;Database=balta;User ID=sa;Password=SUA_SENHA";  `
use =` connectionString = "Server=localhost,1433;Database=balta;User ID=sa;Password=Sua_SENHA; TrustServerCertificate=True"; 

![erro](https://github.com/balta-io/forum/assets/136117631/3d6636bd-522b-4664-87d4-fa007e3b1cbf)
`

**Comment by sinvalfelisberto** [comment, marked as answer]
Valeu!
